rpmbuild/copr_distgit_client.py

```diff
@@ -99,18 +99,28 @@ def _load_config(directory):
 
 def download(url, filename):
     """ Download URL as FILENAME using curl command """
+
+    if not hasattr(download, "curl_has_retry_all_errors"):
```

Comment:
I am very much confused by this. IMHO `download` should be reference to the function itself. So using it this way doesn't make sense to me (same for `download.curl_has_retry_all_errors = ...` below). Am I missing something?

Reply:
This is the way around not using `global` keyword.  A function is object and can have attributes. I'd swear we already have something like this somewhere in our code. Nevermind, if you want me to change this, lemme know.

Reply:
https://stackoverflow.com/questions/279561/what-is-the-python-equivalent-of-static-variables-inside-a-function

Reply:
Oh I see. Thank you for the link.

> Nevermind, if you want me to change this, lemme know.

No, I think it's okay. Can you please just add a short comment saying, that we are checking and setting a static variable for the function?